functests/performance/updating_profile.go

```diff
@@ -133,6 +133,32 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 				}
 			}
 		})
+
+		It("[test_id:22764] verify that realTimeKernel is default when realTimeKernel field does not exist", func() {
+			conditionType := machineconfigv1.MachineConfigPoolUpdating
+
+			// Fail test if realTimeKernel is already installed.
+			for _, node := range workerRTNodes {
+				Expect(execCommandOnWorker(chkKernel, &node)).NotTo(ContainSubstring("PREEMPT RT"))
+			}
+
+			profile.Spec.RealTimeKernel = nil
+			By("Applying changes in performance profile")
+			Expect(testclient.Client.Update(context.TODO(), profile)).ToNot(HaveOccurred())
+
+			if profile.Spec.RealTimeKernel != nil {
+				Expect(err).ToNot(HaveOccurred())
+			} else {
+				By("Checking that the updating MCP status will consistently stay false")
+				Consistently(func() corev1.ConditionStatus {
+					return getMcpStatus(conditionType)
```

Comment:
Just my thought. 
How about changing variable name "conditionType to something more readable, like "conditionUpdating"?

Reply:
Done